DEVELOPER_GUIDE.md

```diff
@@ -146,6 +147,35 @@ Integration tests are automatically run on all pull requests for all supported v
 ### Bulk test runs
 To collect reliability data on test runs there is a manual GitHub action workflow called `Bulk Integration Test`.  The workflow is started for a branch on this project or in a fork by going to [GitHub action workflows](https://github.com/opensearch-project/security/actions/workflows/integration-tests.yml) and selecting `Run Workflow`.
 
+### Checkstyle Violations
```

Comment:
Thanks for adding this section to the README showing how to run the gradle command.